vali_objects/position.py

```diff
@@ -175,19 +175,33 @@ def _handle_liquidation(self, time_ms):
 
         self.close_out_position(time_ms)
 
-    def set_returns(self, realtime_price, net_leverage, time_ms=None):
+    def calculate_return_with_fees(self, current_return_no_fees, net_leverage, trade_pair, time_ms):
+        # Legacy fee calculation
+        if time_ms < 1711496820246:  # TimeUtil.now_in_millis() as of this update
```

Comment:
ahh this is to just impact future miners, not the past ones. So in a month we can fix this to TimeUtil.now_in_millis()

Reply:
So closed position returns will stay static. but let's say a miner closes a position after we merge this change - we will need this timestamp check to ensure that they are using the new logic for fees instead of the legacy logic

Reply:
Actually wait you caught me. I dont think we need a timestamp check since all closed position returns remain static